database/engine/README.md

```diff
@@ -106,4 +106,11 @@ There are explicit memory requirements **per** DB engine **instance**, meaning *
 An important observation is that RAM usage depends on both the `page cache size` and the 
 `dbengine disk space` options. 
 
+## File descriptor requirements
+
+The Database Engine may keep a **significant** amount of files open per instance (e.g. per streaming
+slave or master server). When configuring your system you should make sure there are at least 60
+file descriptors available per `dbengine` instance.
+
```

Comment:
Give them some more info here. Each host streaming data to a master is considered a separate dbengine instance, you need to increase the limit in the systemd service (if not running systemd then where do I do it), mention also that the other things that use fds are the web server and the collector plugins and provide an example sizing (e.g. from stackscale).